lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetDataOutput.java

```diff
@@ -42,6 +43,25 @@ public void writeData(SliceOutput sliceOutput)
         };
     }
 
+    static ParquetDataOutput createDataOutput(ChunkedSliceOutput chunkedSliceOutput)
+    {
+        requireNonNull(chunkedSliceOutput, "chunkedSliceOutput is null");
+        return new ParquetDataOutput()
+        {
+            @Override
+            public int size()
+            {
+                return chunkedSliceOutput.size();
+            }
+
+            @Override
+            public void writeData(SliceOutput sliceOutput)
+            {
+                chunkedSliceOutput.getSlices().forEach(sliceOutput::writeBytes);
```

Comment:
> Avoids separate calls to output stream for writing page headers
and page data

Does it really avoid separate output calls if each chink is written separately?

Reply:
Earlier it was separate slices for page header and data, now that the min chunk size is above page header size (few bytes), we get bigger chunks for output stream.